agent/src/deb/control/control

```diff
@@ -10,6 +10,6 @@ Build-Depends:
  maven (>= 3)
 Standards-Version: 3.9.1
 Architecture: all
-Depends: oracle-jvm-11 (>=11.0.2-1spotify1) | openjdk-11-jre-headless
+Depends: java-11-amazon-corretto-jdk (>=1:11.0.7.10-1) | oracle-jvm-11 (>=11.0.2-1spotify1) | openjdk-11-jre-headless
```

Comment:
The `|` operator for `Depends` is an or operator right? So this indicates that `java-11-amazon-corretto-jdk` should be installed to fulfill the dependency group (strung together by `|`s) but if it's unavailable, either of the other two can be installed?

Reply:
yep that's the intention